Web/RockFood.Api/Controllers/FoodController.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using RockFood.Interfaces;
+using RockFood.Models;
+using RockFood.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace RockFood.Api.Controllers
+{
+    [Route("api/Food")]
+    [ApiController]
+    public class FoodController : ControllerBase
+    {
+        private readonly IFoodService _foodService;
+        public FoodController(IFoodService foodService)
+        {
+            _foodService = foodService;
+        }
+
+        [HttpGet("GetAllFoods")]
```

Comment:
[HttpGet("GetAllFoods")]
->
[HttpGet]

because your action already have this name

remove name from attribute if the same case in other places please

Reply:
I can't because There are two methods Get and if one of them haven't rout swagger is roaring.
We have tolk about it in telegram)))